### PR TITLE
Removed "lol" lol

### PR DESCRIPTION
**Pull request by kholland950**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/kholland950/splittr/pull/29" target="_blank">ReviewNinja

File: index.html [hunk 1]
```diff
@@ -8,7 +8,6 @@
     <title>splittr</title>
 </head>
 <body onload="init()">
-<p class="over">lol</p>
```
Comment (qwoodmansee) [inline index.html:11]: This fixes the issue, good job!

Comment (kholland950) [inline index.html:11]: !completed  !fixed

File: index.html [hunk 2]
```diff
@@ -8,7 +8,6 @@
     <title>splittr</title>
 </head>
 <body onload="init()">
```
Comment (kholland950) [inline index.html:10]: test comment

Comment (kholland950) [inline index.html:10]: !completed

**Pull request merged**